tests/src/model/pages/pods-page.ts

```diff
@@ -62,26 +59,19 @@ export class PodsPage extends BasePage {
     if (await this.pageIsEmpty()) {
       return undefined;
     }
-    const podsTable = await this.getTable();
-    const rows = await podsTable.getByRole('row').all();
-    let first: boolean = true;
-    for (const row of rows) {
-      if (first) {
-        // skip first row (header)
-        first = false;
-        continue;
-      }
-      // test on empty row - contains on 0th position &nbsp; character (ISO 8859-1 character set: 160)
-      const zeroCell = await row.getByRole('cell').nth(0).innerText();
-      if (zeroCell.indexOf(String.fromCharCode(160)) === 0) {
-        continue;
-      }
-      const nameCell = await row.getByRole('cell').nth(3).innerText();
-      const index = nameCell.indexOf(name);
-      if (index >= 0) {
-        return row;
+    try {
+      const podsTable = await this.getTable();
+      const rows = await podsTable.getByRole('row').all();
+      for (let i = rows.length - 1; i > 0; i--) {
+        const nameCell = await rows[i].getByRole('cell').nth(3).getByText(name).count();
```

Comment:
Same for here, ARIA functionality might be a more elegant and stable solution

Reply:
I agree, but this is mostly a refactor of current functionality, I don't want to start changing every locator in the framework in one PR otherwise it will never get merged.

Reply:
Created https://github.com/containers/podman-desktop/issues/5639 to track this effort.